spec/pr_size_checker_spec.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module Danger
+  describe Danger::PRSizeChecker do
+    it 'is a plugin' do
+      expect(described_class.new(nil)).to be_a Danger::Plugin
+    end
+
+    describe 'with Dangerfile' do
+      before do
+        @dangerfile = testing_dangerfile
+        @plugin = @dangerfile.pr_size_checker
+      end
+
+      context 'when checking a PR diff size' do
+        before do
+          allow(@plugin.git).to receive(:added_files).and_return([])
+          allow(@plugin.git).to receive(:modified_files).and_return([])
+          allow(@plugin.git).to receive(:deleted_files).and_return([])
+        end
+
+        shared_examples 'using the default diff size counter without a file selector' do |type|
+          let(:diff_counter_for_type) do
+            type_hash = {
+              insertions: :insertions,
+              deletions: :deletions,
+              all: :lines_of_code
+            }
+
+            type_hash[type]
+          end
+
+          it 'returns a warning when using default parameters in a PR that has larger diff than the default maximum' do
```

Comment:
Nitpick. The plugin doesn't "return" a warning, it "reports" one. Or, even if it does return the warning, that's not what we are testing here when reading `@dangerfile.status_report...`

Reply:
Agree -- I believe I used "report" on another plugin; I'll also make sure to review all of them so they're consistent 👍